### PR TITLE
[XP-3226] Add information about data product templates

### PR DESCRIPTION
**Pull request by gibbok-snowplow**
Notes: We will link this page in the Setup tracking in Setup page of Data Apps.

### CODE REVIEW

File: docs/understanding-tracking-design/defining-the-data-to-collect-with-data-poducts/data-product-templates/index.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+Included event specifications:
+
+* [Page Ping](/docs/collecting-data/collecting-from-own-applications/javascript-trackers/web-tracker/tracking-events/activity-page-pings/)
+* [Page View](/docs/collecting-data/collecting-from-own-applications/javascript-trackers/web-tracker/tracking-events/page-views/)
+
+
+## E-commerce Web
+
+This Data Product template contains all the basic e-commerce interactions that help you analyze customer behavior, identify potential growth opportunities and improve your sales performance.
+
```
Comment (igneel64) [inline docs/understanding-tracking-design/defining-the-data-to-collect-with-data-poducts/data-product-templates/index.md:32]: We need to add:

Tracked using the [Snowplow Ecommerce plugin](https://docs.snowplow.io/docs/collecting-data/collecting-from-own-applications/javascript-trackers/web-tracker/tracking-events/ecommerce/).

File: docs/understanding-tracking-design/defining-the-data-to-collect-with-data-poducts/data-product-templates/index.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+* Refund
+* Remove from cart
+* Track product view
+* Transaction
+* Transaction Error
+
+## Media Web
+
+This Data Product template contains a full set of media tracking interactions to measure video or audio data using the Snowplow Media plugin.
+
```
Comment (igneel64) [inline docs/understanding-tracking-design/defining-the-data-to-collect-with-data-poducts/data-product-templates/index.md:50]: We need to add:

Tracked using the [Snowplow Media plugin](https://docs.snowplow.io/docs/collecting-data/collecting-from-own-applications/javascript-trackers/web-tracker/tracking-events/media/snowplow/).

File: docs/understanding-tracking-design/defining-the-data-to-collect-with-data-poducts/data-product-templates/index.md [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+## Creating Data Product based on Templates using Console
+
+To create a new data product based on a pre-defined template, navigate to the "Data products" section from the navigation bar and click the "Create data product" or "Templates" buttons.
+
+Below is a list of the current templates:
+
+### Base Web Data Product
+
+This Data Product template contains all the tracking of Standard Events. It is recommended to be used in conjunction with JavaScript Trackers (Web and Node.js).
```
Comment (igneel64) [inline docs/understanding-tracking-design/defining-the-data-to-collect-with-data-poducts/data-product-templates/index.md:19]: I would replace:
> JavaScript Trackers (Web and Node.js)

with

> the JavaScript or Browser tracker.

Comment (gibbok-snowplow) [reply docs/understanding-tracking-design/defining-the-data-to-collect-with-data-poducts/data-product-templates/index.md]: done thanks

File: docs/understanding-tracking-design/defining-the-data-to-collect-with-data-poducts/data-product-templates/index.md (hunk 2 above)
Comment (gibbok-snowplow) [reply docs/understanding-tracking-design/defining-the-data-to-collect-with-data-poducts/data-product-templates/index.md:50]: done thanks

File: docs/understanding-tracking-design/defining-the-data-to-collect-with-data-poducts/data-product-templates/index.md (hunk 1 above)
Comment (gibbok-snowplow) [reply docs/understanding-tracking-design/defining-the-data-to-collect-with-data-poducts/data-product-templates/index.md:33]: done thanks

**Review by igneel64 — APPROVED** [review]

**Review by DianaIgnatescu — APPROVED** [review]

**Pull request merged**